minimal-notebook/Dockerfile

```diff
@@ -43,3 +43,6 @@ USER ${NB_UID}
 
 # Add R mimetype option to specify how the plot returns from R to the browser
 COPY --chown=${NB_UID}:${NB_GID} Rprofile.site /opt/conda/lib/R/etc/
+
+# Add setup scripts that may be used by downstream images or inherited images
+COPY setup-scripts/* /usr/bin/local/
```

Comment:
It should be `/usr/local/bin/`, I suppose 🙂
Or, we can use `/opt`.

Reply:
lolol I spent a while trying to figure out why the test broke :D

Reply:
Moved to /opt